src/views/AddItem.jsx

```diff
@@ -76,57 +78,86 @@ export function AddItem({ data, listToken }) {
 
 	return (
 		<>
-			<form onSubmit={submitForm}>
-				<label htmlFor="itemName">
-					Item Name
+			<div className="bg-main text-white w-full h-[80%] absolute bottom-0 md:max-w-md animate-openAddItem overflow-clip z-10">
+				<div
+					id="image"
+					className="h-60 w-full bg-main-darkest absolute bottom-0 animate-appear"
+				/>
+				<form
+					onSubmit={submitForm}
+					className="grid justify-items-center w-full mt-10 relative"
+				>
+					<h1 className="font-bold uppercase text-center my-5">add new item</h1>
+					<label htmlFor="itemName" className="text-center">
+						Item Name
+					</label>
 					<input
 						type="text"
 						id="itemName"
-						value={itemName}
+						value={itemName ? itemName : undefined}
```

Comment:
Are we displaying 'undefined' to the user if there is no itemName? I'm not sure if that's the most user-friendly message?